pkg/storage/minio/minio.go

```diff
@@ -259,7 +259,7 @@ func (c *Client) downloadFile(ctx context.Context, bucket, bucketFolder, file st
 		return nil, ErrArtifactsNotFound
 	}
 
-	if bucketFolder != "" {
+	if bucketFolder != "" && bucketFolder != file {
```

Comment:
In general, nothing is wrong usinf the same name for bucket and file, the problem is appeared when storing logs and artifacts together

Reply:
Thank you for the good comment. As you said, it seems better to separate the log and artifact buckets. Then even if the user sets up to use only one bucket, it seems like testkube need to create a bucket for logs somewhere. Did I understand correctly?

Reply:
yes

Reply:
Good. I will close this PR. Could I contribute this feature if it is not hard for newbie. Could you tell me your opinion about the contribution?